doc/usage/extensions/autodoc.rst

```diff
@@ -344,35 +353,34 @@ There are also new config values that you can set:
    ``'inherited-members'``, ``'show-inheritance'``, ``'ignore-module-all'``
    and ``'exclude-members'``.
 
-   If you set one of these flags in this config value, you can use a negated
-   form, :samp:`'no-{flag}'`, in an autodoc directive, to disable it once.
-   For example, if ``autodoc_default_flags`` is set to ``['members',
-   'undoc-members']``, and you write a directive like this::
+   .. versionadded:: 1.0
 
-      .. automodule:: foo
-         :no-undoc-members:
+   .. deprecated:: 1.8
 
-   the directive will be interpreted as if only ``:members:`` was given.
+      Integrated into :confval:`autodoc_default_options`.
 
-   You can also set `autodoc_default_flags` to a dictionary, mapping option
-   names to the values which can used in .rst files. For example::
+.. confval:: autodoc_default_options
 
-     autodoc_default_flags = {
-        'members': 'var1, var2',
-        'member-order': 'bysource',
-        'special-members': '__init__',
-        'undoc-members': None,
-        'exclude-members': '__weakref__'
-    }
+   The default options for autodoc directives.  They are applied to all autodoc
+   directives automatically.  It must be a dictionally which maps option names
```

Comment:
Sorry I didn't see this earlier but there's a typo: `dictionally` -> `dictionary`

Reply:
thanks!